components/dashboard/src/settings/selectClass.tsx

```diff
@@ -19,11 +19,11 @@ export default function SelectWorkspaceClass(props: SelectWorkspaceClassProps) {
     const { user } = useContext(UserContext);
 
     const [workspaceClass, setWorkspaceClass] = useState<string>(
-        user?.additionalData?.workspaceClasses?.regular || "standard",
+        user?.additionalData?.workspaceClasses?.regular || "g1-standard",
```

Comment:
**question**: Is there a consensus for going with this short-code? If yes, we could surface this in the options below. Otherwise, let's keep it hidden here for now. 🌔

Reply:
The short code is only intended to be used internally.

Reply:
Sounds good. I'm sure in the long run as the options grow and mature, we will have to expose the short code in the interface. Resolving.